src/main/views/cookies.njk

```diff
@@ -185,14 +199,14 @@
       <table class="form-group">
         <thead>
         <tr>
-          <th scope="col">{{ t('Name') }}</th>
-          <th scope="col">{{ t('Purpose') }}</th>
+          <th scope="col">{{ t('Cookie name') }}</th>
+          <th scope="col">{{ t('What this cookie is for') }}</th>
           <th scope="col">{{ t('Expires') }}</th>
         </tr>
         </thead>
         <tbody>
         <tr>
-          <td>TSxxxxxxxx</td>
+          <td>TS01a0d005</td>
```

Comment:
uhh this doesn't exist

Reply:
Its what is in the prototype we had it at one point but apparently not anymore! Ill remove it